nemo/collections/nlp/modules/common/megatron/fused_layer_norm.py

```diff
@@ -57,6 +57,9 @@ def get_layer_norm(hidden_size, eps=1e-5, persist_layer_norm=False):
         persist_layer_norm = False
 
     if persist_layer_norm:
-        return FastLayerNorm(hidden_size, eps)
+        return FastLayerNorm(hidden_size, eps, sequence_parallel_enabled=sequence_parallel)
     else:
+        # TODO: Update when MixedFusedLayerNorm has sequence parallel
+        if sequence_parallel:
+            raise ValueError(f'MixedFusedLayerNorm does not support sequence parallel yet.')
```

Comment:
fyi: updated apex, https://github.com/NVIDIA/apex/pull/1396/files#diff-f779452a51102233a5f08b3621c2fbfe504b4c8d6d313093da7de2917555e022.

I don't think there's any difference between FusedLayerNorm and MixedFusedLayerNorm when we are using them in the context of `torch.cuda.amp`.
```suggestion
```

Reply:
Thanks, I will update this.